doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+TIMING | MALE | YES | Calculate elapsed time and instructions per second statistics for display on START/END messages and STA statistics. tTIMING also displays current z390 and J2SE version on start message and memory usage on ended message.  Note NOTIMING is used in regression tests to force generated files to be identical by using fixed data/time stamp using GregorianCalendar(2005,0,2,22,33,44).  NOTIMING suppresses versions on start message and memory usage on ended message.  NOTIMING also sets NOTIME which prevents application from using time functions..
+TRACE or TRACE(AEGILMPQTV) | E | NO | Turn on ez390 execution trace generation on TRE file and turn off CON.   TRACE will display every instruction executed along with address and value of each operand.  Any combination of the following trace options can be set using the TRACE(...) option.  TRACE(*) will also set TRACEALL.  Note the &SYSTRACE global SETC variable can be used to turn any trace options on or off during execution.
+TRACEA or TRACE(A) | A | NO | Turn on az390 assembly trace generation on TRA file and turn off CON.  TRACEA will display each BAL statement during each pass of the assembler.
+TRACEALL | MALE | NO | Turn on all trace options generating TRM, TRA, TRL, and TRE files and turn off CON.
+TRACEC or TRACE(C) | M | NO | Trace copy file code for TRACEM and TRACEP.  The default is NOTRACEC to eliminated redundant code that usually just contains global variable declarations.
+TRACEG or TRACE(G) | E | NO | Turn on ez390 emulator trace generation on TRE file with trace of GETMAIN/FREEMAIN FQE memory management control block changes and turn off CON.
+TRACEI or TRACE(I) | M | NO | TRACEI will trace each AINSERT showing source on the TRM trace file.
+TRACEL or TRACE(L) | L | NO | Turn on lz390 linker trace generation on TRL file and turn off CON.  TRACEL shows each CSECT, ENTRY, EXTRN and OBJ file being loaded.
+TRACEM or TRACE(M) | M | NO | Turn on mz390 macro processor trace generation on TRM file and turn off CON.  TRACEM displays each conditional macro statement executed plus stored values and AIF compare values.
+TRACEP or TRACE(P) | M | NO | Turn on mz390 macro processor pseudo code generation and execution trace on TRM file and turn off CON.  TRACEP shows each conditional macro pseudo operation performed for each conditional macro statement and the values of all variables.
+TRACEQ or TRACE(Q) | E | NO | Turn on ez390 emulator QSAM/BSAM DCB I/O trace generation on TRE file and turn off CON.
+TRACES or TRACE(S) | M | NO | Displace MLC source lines on console regardless of CON setting to help locate any loops, waits, or hangs in large macro process.  If MCALL is also on, then *MCALL and *MEXIT comments will also be displayed to show location with nested macros.
```

Comment:
Line 119.
1. First sentence. "Displace MLC ..." -> "Displays MLC ...".
2. Last sentence. "... *MCALL and *MEXIT ...". When viewing at adelosa.github.io, the asterisks are removed what was between them is shown in italics. Is there a way to escape the * so that it is considered to be a character and not a markdown delimiter?

Reply:
Fixed 40c73aa